modules/rhel-compute-updating.adoc

```diff
@@ -10,7 +10,7 @@ After you update your cluster, you must update the {op-system-base-full} compute
 
 [IMPORTANT]
 ====
-{op-system-base-full} versions 8.6, 8.7 and 8.8 are supported for {op-system-base} compute machines.
+{op-system-base-full} versions 8.6, 8.7, and 8.8 are supported for {op-system-base} compute machines.
```

Comment:
It seems like the version references in this line were already fixed by the time I started this bug.